packages/react/src/components/AccountSettings/ChangePassword/defaultComponents.tsx

```diff
@@ -12,8 +13,28 @@ export const DefaultCurrentPassword: AccountSettingsPasswordField = (props) => {
   return <PasswordField {...props} />;
 };
 
-export const DefaultNewPassword: AccountSettingsPasswordField = (props) => {
-  return <PasswordField {...props} />;
+export const DefaultNewPassword: AccountSettingsPasswordField = ({
+  validationErrors,
+  ...props
+}) => {
+  return (
+    <>
+      <PasswordField {...props} />
+      <ValidationErrors errors={validationErrors} />
+    </>
+  );
+};
+
+export const DefaultConfirmPassword: AccountSettingsPasswordField = ({
```

Comment:
Looks like we could just have one component, `DefaultPasswordField` and use it in place of `DefaultNewPassword` and `DefaultConfirmPassword`?

Reply:
That's fair -- done in https://github.com/aws-amplify/amplify-ui/pull/2900/commits/5c3906a2728771755681b511968e208cd84cfbf2